pkg/queue/requeue_intervals.go

```diff
@@ -21,10 +21,10 @@ func DetermineRequeueInterval(state shared.State, intervals RequeueIntervals) ti
 		fallthrough
 	case shared.StateProcessing:
 		return intervals.Busy
-	case shared.StateReady:
-		fallthrough
 	case shared.StateWarning:
 		return intervals.Warning
+	case shared.StateReady:
+		fallthrough
```

Comment:
Can you instead of using `fallthrough` here, either return the interval value that is assigned to the state directly or remove the case of that specific state where is anyways handled by the default case then?
Usually `fallthrough` is used to add a piece of logic to the specific case before the fallthrough case, other than that  it's not really needed imo.